### ISSUE TITLE
Use extracted signature along with SBS

### ISSUE THREAD

**Issue by mcdoz2010** [UMCUGenetics/MutationalPatterns#75]
Hi there,
I am finally writing up the results I got with Mutational Patterns since last year. I have NGS data with a lot of artifactual mutations from the polymerase used in PCR, which are interpreted as 3 or 4 different SBS signals. I have a reasonable number of controls amplified from plasmid, and I wondered if it is possible to extract a mutational signal from the controls, and then add this to the SBS signatures, so that when I analyze the real samples, the "PCR artifact" signature gets identified alongisde the "real" signatures in the sample.
Is there any way to do this?

**Comment by FreekManders** [comment]
Yes, you can do this. The mutational signatures are stored inside a matrix. You can combine two matrices using `cbind(m1, m2)`.

**Comment by mcdoz2010** [comment]
Thanks - I will try it. I kind of stupidly just tried m1 + m2 and it didn't work.

**Comment by mcdoz2010** [comment]
Need to extract the column $signatures from nmf_res, then add that to signatures with cbind

**Comment by FreekManders** [comment]
yes, you can extract the signatures with artifact_sigs = nmf_res$signatures and then maybe add colnames using colnames(artifact_sigs) = c("name1", "name2", "name3"). After that you can use cbind.
